packages/adapter-vercel/index.js

```diff
@@ -1,5 +1,6 @@
 const { writeFileSync, mkdirSync, renameSync } = require('fs');
 const { resolve, join } = require('path');
+const { copy } = require('@sveltejs/kit/filesystem');
```

Comment:
Isn't kit using ESM (?) I'm wondering if `adapter-vercel` (cjs) can import the copy fn